cms/static/js/models/license.js

```diff
@@ -0,0 +1,51 @@
+define(["backbone", "backbone.associations"], function(Backbone) {
+    /**
+     * License model used to store the license kind and version
+     * for course and video assets.
+     *
+     * In terms of the kind of licenses, the following options are
+     * used:
+     * ARR      - All rights reserved
+     * CC0      - Creative Commons Zero license (no rights reserved)
+     * CC-BY-*  - Creative Commons Attribution license
+     * CC-*-ND  - Creative Commons NoDerivatives license
+     * CC-*-NC  - Creative Commons NonCommercial license
+     * CC-*-SA  - Creative Commons ShareAlike license
```

Comment:
Is this model written to allow for new, unexpected license types? Or does this model expect that these are the *only* possible licenses that can exist? You should document this one way or the other. If you are writing your model to allow for new, unexpected license types, be sure to write a test case that uses a fake license type and assert that things still work.